core/kamon-core/src/main/resources/META-INF/native-image/io.kamon/kamon-core/reflect-config.json

```diff
@@ -0,0 +1,57 @@
+[
```

Comment:
I'm not really sure of why it works, since the clock is initialized at class-load time. But it does work properly when compiled into a native image :shrug: